src/sdk/GroupSDK.ts

```diff
@@ -42,6 +42,17 @@ export function update(groupID: string, options: GroupUpdateOptions) {
     throw new Error("Group update must provide a new name which is either a non-zero length string or null.");
 }
 
+/**
+ * Rotate a groups private key. This allows for changing the private key of a group without having to re-encrypt any of the documents
+ * encrypted to the group. This operation will re-add every existing admin of the group so depending on the number of admins this
```

Comment:
Might be worth calling out that the group's public key does not change even though that has the same meaning as "without having to re-encrypt any..."

Reply:
Done